CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
+* Update Razor to 9.0.0-preview.24327.6 (PR: [#7273](https://github.com/dotnet/vscode-csharp/pull/7273))
+  * Use a named pipe to communicate projectinfo in vscode (PR: [#10521](https://ryzngard@github.com/dotnet/razor/pull/10521))
```

Comment:
oh interesting... I pulled this from the pr-finder. I wonder if it uses local changes not merged yet since that's the upstream uri I use to avoid multiple account auth. Will fix